docs/services/slam/cartographer/_index.md

```diff
@@ -281,9 +276,7 @@ Watch a map start to appear.
 | `data_dir` | string | **Required** | Path to [the directory](#mapping-modes) used for saving input LIDAR data in <file>/data</file> and output mapping data in <file>/map</file>. |
 | `sensors` | string[] | **Required** | Names of configured RPlidar devices providing data to the SLAM service. |
 | `map_rate_sec` | int | Optional | Rate of <file>/map</file> generation *(seconds)*. <ul> Default: `60`. </ul> |
-| `data_rate_msec` | int | Optional | Rate of <file>/data</file> collection from `sensors` *(milliseconds)*. <ul>Default: `200`.</ul> |
-| `port` | string | Optional | Port for SLAM gRPC server. If running locally, this should be in the form "localhost:<PORT>". If no value is specified a random available port is assigned. |
-| `delete_processed_data` | bool | Optional | <p>Setting this to `true` helps to reduce the amount of memory required to run SLAM.</p> <ul> `true`: sensor data is deleted after the SLAM algorithm has processed it. </ul><ul> `false`: sensor data is not deleted after the SLAM algorithm has processed it. </ul> |
+| `data_rate_msec` | int | Deprecated | Rate of <file>/data</file> collection from `sensors` *(milliseconds)*. <ul>Default: `200`.</ul> |
```

Comment:
@npentrel is this the proper way to mark a config parameter as deprecated?

cc @EshaMaharishi

Reply:
Oh - I think this is the first time we're encountering this. Does it still work? If so this is ok. Will need to think about how we generally handle this but for now this is fine to merge

Reply:
It does still work, but is planned on being removed as part of a future project.